examples/plot_roc.py

```diff
@@ -25,4 +25,6 @@
 print("data_test", data_test.shape)
```

Comment:
Do we still need the list() for assigning values to dataframe column?

Reply:
I redid this example to no longer use dataframes